src/response/chi0.jl

```diff
@@ -291,49 +292,59 @@ Perform in-place computations of the derivatives of the wave functions by solvin
 a Sternheimer equation for each `k`-points. It is assumed the passed `δψ` are initialised
 to zero.
 """
-function compute_δψ!(δψ, basis, H, ψ, εF, ε, δHψ; ψ_extra=[zeros_like(ψk, size(ψk,1), 0) for ψk in ψ],
-                     kwargs_sternheimer...)
+function compute_δψ!(δψ, basis::PlaneWaveBasis{T}, H, ψ, εF, ε, δHψ, ε_minus_q=ε;
+                     ψ_extra=[zeros_like(ψk, size(ψk,1), 0) for ψk in ψ],
+                     q=zero(Vec3{T}), kwargs_sternheimer...) where {T}
+    # We solve the Sternheimer equation
+    #   (H_k - ε_{n,k-q}) δψ_{n,k} = - (1 - P_{k}) δHψ_{n, k-q},
+    # where P_{k} is the projector on ψ_{k} and with the conventions:
+    # - δψ_{k} is the variation of ψ_{k-q}, which implies (for ℬ_{k} the `basis.kpoints`)
+    #     δψ_{k-q} ∈ ℬ_{k-q} and δHψ_{k-q} ∈ ℬ_{k};
+    # - δHψ[ik] = δHψ_{k-q};
```

Comment:
be careful with using `-` for ennumerations here. Could be misinterpreted as a sign.